### PR TITLE
Update csc_assembly.md

### PR DESCRIPTION
**Pull request by siirif**
fi_csc_assembly_v2
We updated our tool assembly. We would also have a new picture, but I don't know how I can upload it ( I tried to attache it here). Thank you for help, if someone can change the pic :)

### CODE REVIEW

**Comment by bianchini88** [comment]
Hi @siirif,

Upload the image to [this folder](https://github.com/elixir-europe/rdmkit/tree/master/images)
and delete the old one if no longer in use.  

After that, update the line below with the new name and/or caption. 
>{% include image.html file="fi_csc_assembly.svg" caption="Figure 1. The CSC - IT Center for Science data management tool assembly." alt="CSC RDMkit" %}

Let me know if you encounter any issue.

**Comment by siirif** [comment]
Hi @bianchini88 ! 
I did not have permission to upload that file into that folder :( I tried (it said uploads are disabled)!

**Comment by bianchini88** [comment]
@siirif Interesting, I was not aware of that. 
You can try to fork the repo, add the image there and then open a PR.

@bedroesb is it intended for contributors to not have permission to upload images?

**Comment by siirif** [comment]
"You can try to fork the repo, add the image there and then open a PR." --> I suck at github and cannot do it :D sorry! plz send help!

**Comment by bianchini88** [comment]
>I suck at github and cannot do it :D sorry! plz send help!

No problem, I will do it. Do I just download the image you sent or is it better if you send me the file? (I suck at graphics and related topics)

**Comment by bedroesb** [comment]
@bianchini88  @siirif Yes, contributors do not have permission to push to master, neither do the editors! https://github.com/siirif/rdm-toolkit/tree/patch-15/images on you branch from where you make the pullrequest, you will be able to do so, normally.

**Comment by bianchini88** [comment]
@bedroesb 
> contributors do not have permission to push to master

Yes, of course, I had misunderstood the issue, sorry (I tend to work locally when dealing with figures).

@siirif 
As Bert says, add the figure at https://github.com/siirif/rdm-toolkit/tree/patch-15/images
and this will automatically update this PR.

**Comment by siirif** [comment]
Okay, I added it: https://github.com/siirif/rdm-toolkit/tree/patch-15/images name: [fi_csc_assembly_v2.svg]
I also tried to change the name in the text file as it was missing the "_v2" from the picture name.

File: pages/tool_assembly/csc_assembly.md [hunk 1]
```diff
@@ -46,24 +46,24 @@ You can access all CSC services through several secure authentication methods. S
...
...uire a data management plan (DMP) as part of the funding application process or after funding has been approved. See e.g. guidance on creating a [DMP for the Research Council of Finland (former Academy of Finland)](https://www.aka.fi/en/research-funding/apply-for-funding/how-to-apply-for-funding/az-index-of-application-guidelines2/data-management-plan/data-management-plan/). There are several tools to guide you through the planning process, such as [DMPTuuli](https://dmptuuli.fi/), a Finnish instance of DMPonline which includes guidance and templates provided by different organisations and funders. DMPs created in DMPTuuli are not yet machine actionable or linked to the CSC data management tool assembly services. However, a DMP is a valuable document when contacting and using the CSC research data management services. Other available DMP tools include [Data Stewardship Wizard](https://researchers.ds-wizard.org/wizard/), [ARGOS](https://argos.openaire.eu/splash/) and [DMPonline](https://dmponline.dcc.ac.uk/). Furthermore, research data support services at Finnish research organisations offer help and guidance through the planning process and when making decisions on data management.
```
Comment (bianchini88) [inline pages/tool_assembly/csc_assembly.md:49]:
```suggestion
Research funders often require a data management plan (DMP) as part of the funding application process or after funding has been approved. See e.g. guidance on creating a [DMP for the Research Council of Finland (former Academy of Finland)](https://www.aka.fi/en/research-funding/apply-for-funding/how-to-apply-for-funding/az-index-of-application-guidelines2/data-management-plan/data-management-plan/). There are several tools to guide you through the planning process, such as [DMPTuuli](https://dmptuuli.fi/), a Finnish instance of DMPonline which includes guidance and templates provided by different organisations and funders. DMPs created in DMPTuuli are not yet machine-actionable or linked to the CSC data management tool assembly services. However, a DMP is a valuable document when contacting and using the CSC research data management services. Other available DMP tools include {%tool "data-stewardship-wizard" %} and {%tool "dmponline" %}. Furthermore, research data support services at Finnish research organisations offer help and guidance through the planning process and when making decisions on data management.
```

File: pages/tool_assembly/csc_assembly.md [hunk 2]
```diff
@@ -46,24 +46,24 @@ You can access all CSC services through several secure authentication methods. S
...
 
-Supercomputers Puhti and Mahti can be used for larger scale analysis and simulations. They will soon be accompanied with the world-class supercomputer {% tool "lumi" %}. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC's computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions. 
+CSC is hosting  EuroHPC's world-class supercomputer [LUMI](https://www.lumi-supercomputer.eu/), which is available for researchers across Europe for projects requiring extreme computing capacity. LUMI is one of the world's most competitive supercomputers and one of the most advanced platforms for artificial intelligence. In addition, you can utilize two national supercomputers Puhti and Mahti for medium to large scale simulations. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC’s computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions.
```
Comment (bianchini88) [inline pages/tool_assembly/csc_assembly.md:59]:
```suggestion
CSC is hosting  EuroHPC's world-class supercomputer [LUMI](https://www.lumi-supercomputer.eu/), which is available for researchers across Europe for projects requiring extreme computing capacity. LUMI is one of the world's most competitive supercomputers and one of the most advanced platforms for artificial intelligence. In addition, you can utilise two national supercomputers Puhti and Mahti for medium to large-scale simulations. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC’s computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions.
```

File: pages/tool_assembly/csc_assembly.md [hunk 3]
```diff
@@ -46,24 +46,24 @@ You can access all CSC services through several secure authentication methods. S
...
+For sensitive data management, there is available [SD Connect](https://research.csc.fi/-/sd-connect) and [SD Desktop](https://research.csc.fi/-/sd-desktop) services. The Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators.  SD Connect allows you to collect, organize and share your encrypted sensitive data in a secure manner via web browser or programmatically.  SD Desktop is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyze the data within a secure cloud. A restricted version of SD Desktop is also available for processing health and social data for secondary use in compliance with the Finnish law and Findata regulation.
```
Comment (bianchini88) [inline pages/tool_assembly/csc_assembly.md:61]:
```suggestion
For management of sensitive data, the [SD Connect](https://research.csc.fi/-/sd-connect) and [SD Desktop](https://research.csc.fi/-/sd-desktop) services are available. The Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators.  SD Connect allows you to collect, organise and share your encrypted sensitive data in a secure manner via web browser or programmatically.  SD Desktop is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyse the data within a secure cloud. A restricted version of SD Desktop is also available for processing health and social data for secondary use in compliance with the Finnish law and Findata regulation.
```

Comment (bianchini88) [reply pages/tool_assembly/csc_assembly.md:61]: >in compliance with the Finnish law and Findata regulation.

Consider adding links for these.

File: pages/tool_assembly/csc_assembly.md [hunk 4]
```diff
@@ -46,24 +46,24 @@ You can access all CSC services through several secure authentication methods. S
...
 
 
 ### Data sharing and publishing
-It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}. Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central European Genome-phenome Archive and the SD Submit at the end of 2021. 
+It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}.  Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central European Genome-phenome Archive.
```
Comment (bianchini88) [inline pages/tool_assembly/csc_assembly.md:65]:
```suggestion
It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}.  Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central European Genome-phenome Archive.
```
```suggestion
It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}.  Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central {%tool "the-european-genome-phenome-archive" %}
```

**Review by bianchini88 — CHANGES_REQUESTED** [review]
Minor spelling/grammar fixes. 

I moved some tools to the new syntax described [here](https://rdmkit.elixir-europe.org/tool_resource_update#what-tool-or-resource-can-be-added-to-the-table).

File: pages/tool_assembly/csc_assembly.md (hunk 3 above)
Comment (siirif) [reply pages/tool_assembly/csc_assembly.md:61]: We have more links on the national page (Finland). But thank you for checking the text (we did try to check with 6 eyes already :D)

Comment (bianchini88) [reply pages/tool_assembly/csc_assembly.md:61]: > We have more links on the national page (Finland). 

You could just say that more info is provided on the national page and link it directly. Up to you.

>But thank you for checking the text

No problem, it was a quick check. The more challenging bit is to move all the existing content to the new syntax for tools. I am trying to do it little by little as I get pages to review.

**Comment by bianchini88** [comment]
@siirif If you are happy with my suggestions, feel free to accept them and ask for a re-review, so we can get the PR merged.

**Comment by siirif** [comment]
Yes thank you @bianchini88 :) looks good to me (I clicked commit to them)!

File: pages/tool_assembly/csc_assembly.md [hunk 5]
```diff
@@ -43,27 +43,27 @@ You can access all CSC services through several secure authentication methods. S
 
 ## For what can you use the CSC data management tool assembly?
 
-{% include image.html file="fi_csc_assembly.svg" caption="Figure 1. The CSC - IT Center for Science data management tool assembly." alt="CSC RDMkit" %}
```
Comment (bedroesb) [inline pages/tool_assembly/csc_assembly.md:46]: if we do not use the old image anymore, I guess I can delete it!

**Review by bianchini88 — APPROVED** [review]

**Pull request merged**